lte/gateway/c/core/oai/test/amf/test_amf_encode_decode.cpp

```diff
@@ -774,7 +775,11 @@ TEST(test_optional_dnn_pdu, test_pdu_session_establish_optional) {
   decode_res = decode_ul_nas_transport_msg(&decode_pdu_sess_est_req, pdu, len);
   EXPECT_EQ(decode_res, true);
   // build uplinknastransport
-  EXPECT_EQ(dnn, decode_pdu_sess_est_req.dnn.dnn);
+  // EXPECT_EQ(dnn, decode_pdu_sess_est_req.dnn.dnn);
```

Comment:
Remove commented code and irrelevant comment in line 777

Reply:
addressed review comment